testsuite/tests/kuadrant/authorino/metrics/conftest.py

```diff
@@ -1,6 +1,9 @@
 """Conftest for the Authorino metrics tests"""
 import pytest
+import yaml
```

Comment:
nit: should be a top-level import

Reply:
If it is ok with Black, I will probably not change it